.github/workflows/evaluation.yml

```
@@ -84,6 +148,40 @@ jobs:
           name: LLVM evaluation
           path: bv-evaluation/results
 
+  evaluation-alive-hdel:
+    name: Evaluation Alive and Hacker's Delight
+    runs-on: ubuntu-latest
+    needs: eval-build
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Load Lean and Lean-MLIR from Cache
+        id: cache-lake
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            ~/.elan
+            .lake/packages
+            .lake/build
+            TacBench/.lake
+          key: ${{ runner.os }}-lake-packages-${{ hashFiles('lake-manifest.json') }}
```

```suggestion
          key: ${{ runner.os }}-lake-packages-${{ hashFiles('**/lake-manifest.json') }}
```

While this is extra-paranoid, I think it's cleaner to hash by all manifests.

---

Are the manifests of dependencies even respected in a `lake build`? I thought not.

Regardless, it's not actually a soundness concern, since lake will do its own verification of the cache, so this is not defensive paranoia; rather it's premature optimization and I'd suggest we stick to the old version. (But, I don't feel super strongly about it, so take it or leave it)

---

Happy to discuss this in person, to go either way. It seems none is a critical concern at the moment, so I merge this as is and can then address this in a follow-up conversation tomorrow.